optimade/server/entry_collections/mongo.py

```diff
@@ -53,13 +59,10 @@ def __init__(
         self._check_aliases(self.resource_mapper.all_aliases())
         self._check_aliases(self.resource_mapper.all_length_aliases())
 
-    def __len__(self):
+    def __len__(self) -> int:
         return self.collection.estimated_document_count()
 
-    def __contains__(self, entry):
-        return self.collection.count_documents(entry.dict()) > 0
-
-    def count(self, **kwargs):
+    def count(self, **kwargs) -> int:
```

Comment:
Here there is an opportunity to add a docstring laying out the allowed values for `kwargs`.